src/chrome/content/rules/ESOMAR.org.xml

```diff
@@ -1,26 +1,42 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://esomar.org/ => https://esomar.org/: (60, 'SSL certificate problem: unable to get local issuer certificate')
-	Fully covered subdomains:
-
-		- (www.)
-		- directory
-		- payment
-		- rwconnect
+	Invalid certificate:
+		ana.esomar.org
+		www.ana.esomar.org
+		associations.esomar.org
+		www.directory.esomar.org
+		extranet.esomar.org
+		www.rating.esomar.org
+		survey.esomar.org
+		wiki.esomar.org
+
+	Login required:
+		cloud.esomar.org
+		library.esomar.org
+		tracker.esomar.org
+
+	Different content http/https:
+		jobs.esomar.org
+		my.esomar.org
+
+	Mixed content:
```

Comment:
I have mixed content from `wyldedragonne.com` and `connect.facebook.com` on https://rwconnect.esomar.org/ with Chromium.